spring-integration-file/src/main/java/org/springframework/integration/file/filters/CompositeFileListFilter.java

```diff
@@ -42,7 +42,7 @@
  */
 public class CompositeFileListFilter<F> implements ReversibleFileListFilter<F>, ResettableFileListFilter<F>, Closeable {
 
-	private final Set<FileListFilter<F>> fileFilters;
+	protected final Set<FileListFilter<F>> fileFilters;
```

Comment:
Right it's bogus but I don't see a way on Sonar to suppress only for final fields.